src/routes/docs/tutorials/sveltekit-ssr-auth/step-2/+page.markdoc

```diff
@@ -0,0 +1,53 @@
+---
+layout: tutorial
+title: Create project
+description: Add Authentication to a SvelteKit project using Appwrite.
+step: 2
+---
+
+
+Create a Svelte project using [SvelteKit](https://kit.svelte.dev/docs/creating-a-project).
+
+```sh
+npm create svelte@latest
+```
+
+The command will give you a prompt with several project types. We'll be starting with a skeleton project.
+
+The prompt will be something similar to this.
+
+```sh
+create-svelte version 3.2.0
+
+┌  Welcome to SvelteKit!
+│
+◇  Where should we create your project?
+│  my-svelte-project
+│
+◇  Which Svelte app template?
+│  Skeleton project
+│
+◇  Add type checking with TypeScript?
+│  Yes, using JavaScript with JSDoc comments
+│
+◇  Select additional options (use arrow keys/space bar)
+│  Add ESLint for code linting
+│
+└  Your project is ready!
+```
+
+After the prompt is finished, you can head over to the newly create project.
+
+```sh
+cd my-svelte-project
+npm install
+```
+
+## Adding `node-appwrite` to Your Svelte App
```

Comment:
Please use `#` instead of `##` because internal logic demotes tags by one level.

Also check for tags where you don't have {% #link-to-header %}

Reply:
Done